doc/kube-api-test.md

```diff
@@ -0,0 +1,153 @@
+# kube-api-test
```

Comment:
This document needs to be referenced in root README.md as well . Maybe we should add it below [`Mocking Kubernetes`](https://github.com/fabric8io/kubernetes-client?tab=readme-ov-file#mocking-kubernetes) section?

Reply:
added a section into the docs, pls check if this is ok this way. thx!